src/__tests__/to-contain-html.js

```diff
@@ -0,0 +1,64 @@
+import {render} from './helpers/test-utils'
+
+test('.toContainHTML', () => {
+  const {queryByTestId} = render(`
+    <span data-testid="grandparent">
+      <span data-testid="parent">
+        <span data-testid="child"></span>
+      </span>
+      <svg data-testid="svg-element"></svg>
+    </span>
+    `)
+
+  const grandparent = queryByTestId('grandparent')
+  const parent = queryByTestId('parent')
+  const child = queryByTestId('child')
+  const nonExistantElement = queryByTestId('not-exists')
+  const fakeElement = {thisIsNot: 'an html element'}
+  const stringChildElement = '<span data-testid="child"></span>'
+  const nonExistantString = '<span> Does not exists </span>'
```

Comment:
(nit-picking alert!) should say "Does not exist" without the final "s".

Also, I still do not see any test about what happens if you pass invalid html, as in `<div>hello</p>`. I'm still not clear if in that case it fails because it won't find the invalid html in the container, or simply because it will fail to parse the invalid html in the first place.